apps/ensadmin/src/app/error.tsx

```diff
@@ -0,0 +1,34 @@
+"use client";
+
+import { ENSNodeErrorMessage } from "@/components/ensnode/components";
+import { ErrorMessage } from "@/components/ui/error-message";
+import { useEffect } from "react";
+
+interface ErrorProps {
+  error: Error;
+  reset: () => void;
+}
+
+/**
+ * Renders an error message for unknown errors at the root of the app.
```

Comment:
```suggestion
 * Renders an error message for any errors at the root of the app.
```

Is that right? And is it correct to really say we render it at the literal "root" of the app? Is this 100% technically correct that it's really at the root of the whole app?

Reply:
I believe it is correct, it's the `app/error.tsx`. If `app/` is the application root, then the `app/error.tsx` file is in the root of the whole app.

Next.js uses `app/` directory an an entrypoint to all app routes.